docs/UserGuide.md

```diff
@@ -4,16 +4,128 @@ title: "User Guide"
 pageNav: 3
 ---
 
-# AB-3 User Guide
+# TrackMate User Guide
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+## Welcome to the TrackMate User Guide
 
-<!-- * Table of Contents -->
-<page-nav-print />
+Welcome to the **TrackMate User Guide** - your essential supplementary tools in your teaching journey as a SOC
+Teaching Assistant at National University of Singapore (NUS).
+
+In this comprehensive user guide, we will take you to experience a full journey with TrackMate step by step.
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Table of Content
+
+[1. Introduction](#introduction)
+- [1.1 What is TrackMate-NUS](#what-is-trackmate)
+- [1.2 User Proficiency and Expectations](#user-proficiency-and-expectations)
+- [1.3 Why This Guide Matters](#why-this-guide-matters)
+
+[2. How to use this User Guide](#how-to-use-this-user-guide)
+- [2.1 Navigating the Document](#navigating-the-document)
+- [2.2 Sections](#sections)
+- [2.3 Icons](#icons)
+
+[3. Getting Started](#getting-started)
+- [3.1 Installation](#installation)
+- [3.2 Graphical User Interface Layout](#graphical-user-interface-layout)
+    - [3.2.1 User Interface Overview](#user-interface-overview)
+    - [3.2.2 Additional UI Components](#additional-ui-components)
+- [3.3 How to use TrackMate commands](#how-to-use-track-mate-nus-commands)
+    - [3.3.1 Parameter Prefixes](#parameter-prefixes)
+    - [3.3.2 Parameters](#parameters)
+    - [3.3.3 Command Format](#command-format)
+
+[4. Commands](#commands)
```

Comment:
I really like how you organized the commands into different categories. It makes the document much easier to navigate and understand!